src/features/dataset/state/datasetState.ts

```diff
@@ -18,6 +21,14 @@ export const SET_BODY_LOADING = 'SET_BODY_LOADING'
 
 export const RESET_DATASET_TITLE_ERROR = 'RESET_DATASET_TITLE_ERROR'
 
+export const SET_DATASET_IS_EDITABLE = 'SET_DATASET_IS_EDITABLE'
+
+export const SET_DATASET_META = 'SET_DATASET_META'
+
+export const SET_DATASET_README = 'SET_DATASET_README'
+
+export const SET_DATASET_COMMIT_TITLE = 'SET_DATASET_COMMIT_TITLE'
+
```

Comment:
I think once we get rid of the actions in the above file, we can probably get rid of these as well.